src/http.zig

```diff
@@ -3156,11 +3168,10 @@ fn handleResponseBodyFromSinglePacket(this: *HTTPClient, incoming_data: []const
         this.state.total_body_received += incoming_data.len;
     }
     defer {
-        if (this.progress_node) |progress| {
-            progress.activate();
-            progress.setCompletedItems(incoming_data.len);
-            progress.context.maybeRefresh();
-        }
+        this.downloaded_bytes += incoming_data.len;
+
+        if (this.progress_node != null)
+            updateProgress(this);
```

Comment:
```suggestion
            this.updateProgress();
```

tip: method call syntax is equivalent of a function who's first argument is the base

Reply:
right I forgot about that 😅